src/main/java/edu/ucsb/ucsbcslas/controllers/TutorNotesController.java

```diff
@@ -0,0 +1,274 @@
+        Set<String> courseNumbers = new HashSet<String>();
+        for (TutorNotes ta : tutorNotes) {
+            courseNumbers.add(ta.getCourse().getName());
+
+        }
+
+        ObjectMapper mapper = new ObjectMapper();
+        String body = mapper.writeValueAsString(courseNumbers);
+        return ResponseEntity.ok().body(body);
+    }
+
+    @GetMapping(value = "/api/member/tutorNotes/byCourseNumber/{courseNumber}", produces = "application/json")
+    public ResponseEntity<String> getTutorNotesByCourseID(@PathVariable("courseNumber") String courseNumber)
+            throws JsonProcessingException {
+        List<TutorNotes> tutorNotes = tutorNotesRepository.findAll();
+        List<TutorNotes> tutorNotesMatchingCourse = new ArrayList<TutorNotes>();
+        for (TutorNotes ta : tutorNotes) {
+            if (ta.getCourse().getName().equals(courseNumber)) {
+                tutorNotesMatchingCourse.add(ta);
+            }
+
+        }
+        ObjectMapper mapper = new ObjectMapper();
+        String body = mapper.writeValueAsString(tutorNotesMatchingCourse);
+        return ResponseEntity.ok().body(body);
+    }
+
+    @GetMapping(value = "/api/member/tutorNotes/byTutor/{tutor}", produces = "application/json")
+    public ResponseEntity<String> getTutorNotesByTutor(@PathVariable("tutor") String tutorEmail)
+            throws JsonProcessingException {
+
+        Optional<Tutor> tutor = tutorRepository.findByEmail(tutorEmail);
+        List<TutorNotes> tutorNotes = new ArrayList<TutorNotes>();
+        tutorNotes = tutorNotesRepository.findAllByTutor(tutor.get());
+        ObjectMapper mapper = new ObjectMapper();
+        String body = mapper.writeValueAsString(tutorNotes);
+        return ResponseEntity.ok().body(body);
+
+    }
+
+    @GetMapping(value = "/api/member/tutorNotes/{id}", produces = "application/json")
+    public ResponseEntity<String> getTutorNotes(@PathVariable("id") Long id) throws JsonProcessingException {
+        Optional<TutorNotes> tutorNotes = tutorNotesRepository.findById(id);
+        if (tutorNotes.isEmpty()) {
+            return ResponseEntity.notFound().build();
+        }
+
+        ObjectMapper mapper = new ObjectMapper();
+        String body = mapper.writeValueAsString(tutorNotes.get());
+        return ResponseEntity.ok().body(body);
+    }
+
+    @PutMapping(value = "/api/member/tutorNotes/{id}", produces = "application/json")
+    public ResponseEntity<String> updateTutorNotes(@RequestHeader("Authorization") String authorization,
+            @PathVariable("id") Long id, @RequestBody @Valid String incomingTutorNotes) throws JsonProcessingException {
+        if (!authControllerAdvice.getIsMember(authorization)) {
+            return new ResponseEntity<>("Unauthorized Request", HttpStatus.UNAUTHORIZED);
+        }
+        if (!authControllerAdvice.getIsAdmin(authorization)) {
+              return getUnauthorizedResponse("admin");
+            }
```

Comment:
Similar question to above- as far as I'm aware, member endpoints only require the `isMember` check. But let me know if this is the result of a discussion about these endpoints!